sandbox.c

```diff
@@ -128,7 +129,17 @@ open_landlock(void)
 					LANDLOCK_ACCESS_FS_MAKE_REG,
 	};
 
-	return landlock_create_ruleset(&attr, sizeof(attr), 0);
+	fd = landlock_create_ruleset(&attr, sizeof(attr), 0);
+	if (fd == -1) {
+		switch (errno) {
+		case ENOSYS:
+		case ENOTSUPP:
+			return -1;
+		default:
+			err("can't create landlock ruleset");
```

Comment:
`error: too few arguments to function 'err'`

Reply:
sorry, pushed to fast